DataFormats/DeepFormats/src/classes_def.xml

```diff
@@ -0,0 +1,18 @@
+<lcgdict>
+
+  <class name="deep::JetFeatures"/>
+  <class name="deep::SecondaryVertexFeatures"/>
```

Comment:
it looks like there is intent to save these classes (as a part of DeepFlavourTagInfo) in the event.
classVersion should be added for schema evolution tracking.

Reply:
added in https://github.com/cms-sw/cmssw/pull/19893/commits/dcc93323f80ff2a7ea359c87d628bade95915b1d